HardcodeTray/modules/applications/js.py

```diff
@@ -0,0 +1,76 @@
+"""
+Fixes Hardcoded tray icons in Linux.
+
+Author : Bilal Elmoussaoui (bil.elmoussaoui@gmail.com)
```

Comment:
Pretty sure I didn't wrote this file, put your name instead

Reply:
Oops, copy-paste. ;)